pkg/compute/testdata/build/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "edge-compute-default-rust-template"
+version = "0.1.0"
+authors = ["phamann <patrick@fastly.com>"]
+edition = "2018"
```

Comment:
👀

Reply:
This is the correct edition. See here for more info: https://doc.rust-lang.org/edition-guide/editions/index.html

Reply:
Oh, it's the Rust edition, I see 👍